OneSignalSDK/onesignal/src/main/java/com/onesignal/ADMMessageHandler.java

```diff
@@ -43,6 +44,18 @@
    public static class Receiver extends ADMMessageReceiver {
       public Receiver() {
          super(ADMMessageHandler.class);
+         boolean ADMLatestAvailable = false;
+         try{
+            Class.forName( "com.amazon.device.messaging.ADMMessageHandlerJobBase" );
+            ADMLatestAvailable = true ;
+         }
+         catch (ClassNotFoundException e)
+         {
+            // Handle the exception.
+         }
+         if (ADMLatestAvailable) {
+            registerJobServiceClass(ADMMessageHandlerJob.class, 123891);
```

Comment:
if the number is the job id, can you make it a constant? usually, ids started with a number to identify the job type, for example for intents we use 100x, 200x, or 10x, 20x, for this ADM we could use 3000? just to have a convention regarding all ADM to start with 300.

Reply:
I agree on making this a constant.

We don't want to use something like 3000 as this is something an app developer might pick for something else. The value `123891` is fine, we want to make sure it is unique.